lib/mistral_ai_llm_controller_example/mistral_ai_llm_controller_page.dart

```diff
@@ -0,0 +1,212 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/logger_dialog.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/model.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/prompt.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/utils.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/mistral_client.dart';
+import 'package:mistralai_client_dart/mistralai_client_dart.dart';
+
+class MistralAiLlmControllerPage extends StatefulWidget {
```

Comment:
just to be consisten with everything else
```suggestion
class MistralAILlmControllerPage extends StatefulWidget {
```

Reply:
Resolved in [d4b497c](https://github.com/nomtek/flutter_ai_examples/pull/6/commits/d4b497c5984b8e61d0fc1da971796c5191c74f5b) and [913da46](https://github.com/nomtek/flutter_ai_examples/pull/6/commits/913da461c6f57b736aaa580a00e78113e08fa4c7)